business_objects/general.py

```diff
@@ -53,8 +53,8 @@ def flush_or_commit(commit: bool = False) -> None:
         session.flush()
 
 
-def execute(sql: str) -> Any:
-    return session.execute(sql)
+def execute(sql: str, *args) -> Any:
+    return session.execute(sql, *args)
```

Comment:
usually we use the execute method with a string object.
If I found it correctly the *arg were introduced for the update thing which doesn't use a string anymore. Maybe we should use a different method for this so there is no confusion in the future?

e.g.:`execute_query(query:<type of query object>, *args)`

Reply:
The args could become handy for other executions, too. Of course i can add a dedicated function for this but the args are per default just optional and can be ignored if not needed

Reply:
Changed type of sql from str to Any